lib/portlayer/storage/vsphere/container.go

```diff
@@ -275,7 +310,10 @@ func (c *ContainerStore) Export(op trace.Operation, id, ancestor string, spec *a
 		return nil, errors.New("mismatched datasource types")
 	}
 
-	tar, err := archive.Diff(op, fl.Name(), fr.Name(), spec, data)
+	// if we want data, exclude the xattrs, otherwise assume diff
+	xattrs := !data
```

Comment:
Yeah, I did that so people coming along later reading this wouldn't be puzzled as to why.